packages/relay-compiler/core/__tests__/__snapshots__/GraphQLIRVisitor-test.js.snap

```diff
@@ -79,10 +79,10 @@ fragment UserFragment_mutated on User @argumentDefinitions(
   ... on Mutated @include(if: $cond_mutated) {
     id_mutated
     __typename_mutated
-    checkins_mutated(environments_mutated: [WEB_mutated]) {
+    checkins_mutated(environments_mutated: [MOBILE]) {
       __typename_mutated
     }
-    friends_mutated(after_mutated: $after_mutated, first_mutated: $first_mutated, traits_mutated: [HELPFUL_mutated]) {
+    friends_mutated(after_mutated: $after_mutated, first_mutated: $first_mutated, traits_mutated: [DERISIVE]) {
```

Comment:
use positive language? :-)

Reply:
I can't, serialization for enums only returns valid values and HEPLFUL_mutated isn't a valid enum value for traits.